nucypher-core/src/access_control.rs

```diff
@@ -22,6 +21,8 @@ pub struct AuthenticatedData {
     pub conditions: Option<Conditions>,
 }
 
+impl Eq for AuthenticatedData {}
```

Comment:
Does it work if you just add it to the list in `#[derive]`?

Reply:
Not quite. We'd also need to `#[derive(Eq)]` for `DkgPublicKey` in `ferveo`